frontend/packages/integration-tests-cypress/tests/crud/rolebindings.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { checkErrors, testName } from '../../support';
+import { nav } from '../../views/nav';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+import { detailsPage } from '../../views/details-page';
+import { roleBindings } from '../../views/rolebindings';
+import { errorMessage } from '../../views/form';
+
+describe('RoleBindings', () => {
+  const bindingName = `${testName}-cluster-admin`;
+  const roleName = 'cluster-admin';
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.switcher.changePerspectiveTo('Administrator');
+    nav.sidenav.switcher.shouldHaveText('Administrator');
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  it(`displays "Create RoleBinding" page`, () => {
```

Comment:
It looks like this entire test is covered below in the next test. Is it needed?

Reply:
I wasn't sure so I left that one alone. Do you think this is ok to drop @dtaylor113?

Reply:
yes, it's just a repeat of the test below.l

Reply:
yes, i'd drop it, it's a duplicate/subset of the test below.

Reply:
Thanks; will drop!